### PR TITLE
Converting rest of Tables to Grids

### PR DESCRIPTION
**Pull request by s-laugh**
replicating from what was done to `codeTable.html`

Continuing from #1316 #1318

### CODE REVIEW

**Comment by s-laugh** [comment]
Looking for review/merge.. @nschonni @smellems

File: _includes/codeTable.html [hunk 1]
```diff
@@ -207,7 +207,7 @@
         {%- for admin in govLevels[1] -%}
           {%- for release in admin[1].releases -%}
           {%- unless release.tags.en contains site.excludeTag -%}
-          <tr class="col-xs-12 col-sm-6 col-md-4" style="vertical-align: top; min-height: 349px;">
+          <tr class="col-xs-12 col-sm-6 col-md-4" style="vertical-align: top; min-height: 350px;">
```
Comment (smellems) [inline _includes/codeTable.html:210]: Any idea why the equal height isn't working?

Looking at the Style Guide, I see you don't need the style attribute here.  WET will add it with the min-height, but not the right one.  Even if I remove it it still does'nt work

**Comment by smellems** [comment]
I like the changes, maybe we could still merge like this and fix height with another PR

File: _includes/codeTable.html (hunk 1 above)
Comment (s-laugh) [reply _includes/codeTable.html:210]: I think the equal height doesn't work due to where the box is created - I'll take a closer look

Also can double check the inline styling to remove it where it's not needed.

**Comment by s-laugh** [comment]
> ix height with an

I would be happy to do that!

File: _includes/codeTable.html (hunk 1 above)
Comment (s-laugh) [reply _includes/codeTable.html:210]: Opened issue to be resolved in new PR

**Review by smellems — APPROVED** [review]

**Comment by smellems** [comment]
Looks like I'm no longer authorized to merge :smile:  @nschonni

**Comment by nschonni** [comment]
@smellems you had Write access, but I'm changed it to "Maintain" to see if you get the merge button back

**Pull request merged**